src/pages/UserList/ContextMenu/Modals/ResetTwoFAModal.js

```diff
@@ -47,45 +47,39 @@ const Disable2FaModal = ({ user, refetchUsers, onClose }) => {
     }
 
     return (
-        <Modal>
-            <ModalTitle>
-                {i18n.t(
-                    'Disable two factor authentication for user {{- name}}',
-                    {
-                        name: user.displayName,
-                    }
-                )}
-            </ModalTitle>
+        <Modal small>
```

Comment:
Just made this small as it fits the pattern in the password reset, but I don't really care either way